Bug on /events command

```
[2024-12-10T15:53:00.350] [INFO] default - Notification sent for user 807978815267012629 at Tue Dec 10 2024 15:53:00 GMT+0000 (Coordinated Universal Time) for prayer Maghrib located at Domont, France 
    at async module.exports.run (/usr/src/bot/commands/religion/events.js:23:29)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at module.exports.AladhanAPI.getIslamicUpcomingEvents (/usr/src/bot/utils/aladhan_api.js:99:19)
[2024-12-10T15:52:08.199] [ERROR] default - TypeError: date.getMonth is not a function
    at WebSocketShard.emit (/usr/src/bot/node_modules/@vladfrangu/async_event_emitter/dist/index.cjs:287:31)
    at WebSocketShard.<anonymous> (/usr/src/bot/node_modules/@discordjs/ws/dist/index.js:1190:51)
    at WebSocketManager.emit (/usr/src/bot/node_modules/@vladfrangu/async_event_emitter/dist/index.cjs:287:31)
    at WebSocketManager.<anonymous> (/usr/src/bot/node_modules/discord.js/src/client/websocket/WebSocketManager.js:232:12)
    at WebSocketManager.handlePacket (/usr/src/bot/node_modules/discord.js/src/client/websocket/WebSocketManager.js:348:31)
    at module.exports [as INTERACTION_CREATE] (/usr/src/bot/node_modules/discord.js/src/client/websocket/handlers/INTERACTION_CREATE.js:4:36)
    at InteractionCreateAction.handle (/usr/src/bot/node_modules/discord.js/src/client/actions/InteractionCreate.js:97:12)
    at Client.emit (node:events:520:28)
    at Client.<anonymous> (/usr/src/bot/events/handle_interactions.js:8:16)
    at module.exports.trackCommandActivities (/usr/src/bot/utils/track_commands_activities.js:17:75)
[2024-12-10T15:52:07.268] [WARN] default - TypeError: Cannot read properties of null (reading 'id')
```

--- a/apis/aladhan_api.js
+++ b/apis/aladhan_api.js
@@ -94,7 +94,7 @@ module.exports.AladhanAPI = class {
       const events = [];
       let calendar = await this.getIslamicCalendarByYear(today.getFullYear());
 
-      for (let date = today; date < endDate; date = date.setMonth(date.getMonth() + 1)) {
+      for (let date = today; date < endDate; date.setMonth(date.getMonth() + 1)) {
          // If the month is Janurary, get the next year's calendar
          if (date.getMonth() === 0) { // 0 is January
             calendar = await this.getIslamicCalendarByYear(date.getFullYear());

--- a/changelog.md
+++ b/changelog.md
@@ -143,7 +143,7 @@ If you'd like to contributing to the project, head over to my [Github](https://g
   
 ---
 
-## [1.2.10] - 2024-12-18
+## [1.2.10] - 2024-12-20
 
 Hello brothers and sisters, I am happy to announce a new release on MuslimBot. 🙂
 
@@ -154,6 +154,9 @@ If you like to contributing to the project, head over to my [Github](https://git
 - Add link to release notes in the changelog sent on discord
 - Create 2 templates (feature,bug) for github issues
 - Autocomplete for commands: `/prayer`, `/subscribe` and `/unsubscribe` . It will help the user to find the city and country they want to subscribe to.
+- Create a donation link and embed it in few messages
+- Fix commands_statistics sql view
+- Fix `/events` command
 
 ## Changed
 - Move Docker Image to Github Container Registry

--- a/commands/_general/vars.js
+++ b/commands/_general/vars.js
@@ -1,12 +1,15 @@
+const packageJson = require('../../package.json');
+
+const paypalDonationUrl = 'https://www.paypal.com/donate/?hosted_button_id=T94D4G4PY3YT8';
+
 module.exports = {
-   primaryColor: '#002900',
-   greenColor: '#009000',
+   primaryColor: '#FFD700',
+   greenColor: '#008000',
    redColor: '#DB0109',
-   errorColor: '#DB0109',
-   reecoom: 'https://www.nautiljon.com/images/perso/00/15/reecom_5851.jpg',
+   errorColor: '#FF0000',
    githubUrl: 'https://github.com/fivekage/muslimbot',
    topggUrl: 'https://top.gg/bot/1183399354166415481',
-   paypalDonationUrl: 'https://www.paypal.com/donate/?hosted_button_id=T94D4G4PY3YT8',
+   paypalDonationUrl: paypalDonationUrl,
    prayerGif: 'https://i.giphy.com/12ihpr4WmwKJsQ.gif',
-   footerText: `${require('../../package.json').version} - MuslimBot 🕋 - For any help type /help command`
+   footerText: `${packageJson.version} - MuslimBot 🕋 - For any help type /help command)`
 };

--- a/commands/religion/events.js
+++ b/commands/religion/events.js
@@ -13,7 +13,7 @@ module.exports.help = {
 
 module.exports.run = async (_client, interaction) => {
 
-   await interaction.deferReply({ ephemeral: true });
+   await interaction.deferReply();
    const aladhanAPIObj = new AladhanAPI(3);
 
    try {

--- a/commands/religion/subscriptions.js
+++ b/commands/religion/subscriptions.js
@@ -59,14 +59,14 @@ module.exports.run = async (_client, interaction) => {
          ...enabledSubscriptions.map((subscription) => {
             return {
                name: `${subscription.name}`,
-               value: `Currently ${subscription.enabled ? '**enabled**' : '*disabled*'}, created at ${subscription.createdAt}`,
+               value: `Currently **enabled**, created at ${subscription.createdAt}`,
                inline: false,
             };
          }),
          ...disabledSubscriptions.map((subscription) => {
             return {
-               name: `${subscription.name}`,
-               value: `Currently ${subscription.enabled ? '**enabled**' : '*disabled*'}, created at ${subscription.createdAt}`,
+               name: `~~${subscription.name}~~`,
+               value: `Currently disabled, created at ${subscription.createdAt}`,
                inline: false,
             };
          }),

--- a/events/handle_newguild.js
+++ b/events/handle_newguild.js
@@ -73,6 +73,10 @@ module.exports.handleNewGuild = async (client) => {
                name: 'Support',
                value: 'If you have any questions or suggestions, please send me a message <@317033647045607424>)',
             },
+            {
+               name: 'Donation',
+               value: `You can [donate](${vars.paypalDonationUrl}) to support the development of MuslimBot`,
+            }
          ])
          .setFooter({ text: `${require('../package.json').version} - MuslimBot 🕋 - For any help type /help command` });
 

--- a/utils/sync_version.js
+++ b/utils/sync_version.js
@@ -39,7 +39,7 @@ module.exports = {
                               .setAuthor({ name: 'MuslimBot' })
                               .addFields([
                                  { name: 'Thank you 🙏', value: `If you like the bot, please leave a vote and a review on: [Top.gg](${vars.topggUrl})` },
-                                 { name: 'Donation', value: `If you want to support the bot, you can donate on [PayPal](${vars.paypalDonationUrl})` },
+                                 { name: 'Donation', value: `If you want to support the project, you can make a [Donation](${vars.paypalDonationUrl})` },
                                  { name: 'Support', value: `If you have any questions or need help, please open an issue on [GitHub](${vars.githubUrl}/issues)` },
                               ])
                               .setColor(vars.primaryColor)

